docs/development.md

```diff
@@ -38,53 +77,20 @@ npm run type-check
 
 ```
 
-## Make scripts
-
-Podman is a requirement. Install and init instructions [here](https://podman.io/docs/installation).
+UI stack supports two mode of deployments:
 
-```bash
-# Run markdown linter
-make md-lint
-```
+- github - This is the default mode and it allows users to push their knowledge and skill contribution to the github taxonomy repository.
+- native - This mode allow users to keep the skill and knowledge contribution in their local machine.
 
-## Deploying the UI stack on your local machine
+## Running the UI in Native Deployment Mode
 
-Set the .env in the ui directory and make sure you uncomment the `IL_UI_DEPLOYMENT=dev`. Once .env file is setup, run the following:
+To enable the native mode, set the `IL_UI_DEPLOYMENT=native` in the .env file. Once the flag is set, the UI will not push the knowledge and skill contribution to the github repository. Instead, it will keep the contribution in the local machine. In the `native` mode, the UI login page will show username and password input box to authenticate the user. You can setup the username and password in the .env file through the `IL_UI_ADMIN_USERNAME` and `IL_UI_ADMIN_USERNAME` flags.
```

Comment:
```suggestion
To enable the native mode, set the `IL_UI_DEPLOYMENT=native` in the .env file. Once the flag is set, the UI will not push the knowledge and skill contribution to the github repository. Instead, it will keep the contribution in the local machine. In the `native` mode, the UI login page will show username and password input box to authenticate the user. You can setup the username and password in the .env file through the `IL_UI_ADMIN_USERNAME` and `IL_UI_ADMIN_PASSWORD` flags.
```

Reply:
@Gregory-Pereira thanks for catching this, will fix it in follow-up PR.